docs/efun/contrib/store_variable.md

```diff
@@ -28,6 +31,6 @@ title: contrib / store_variable.pre
 
     store_variable( "weight", 150, this_player() ) ;
 
-### SEE ALSO
+## SEE ALSO
```

Comment:
I think you accidentally deleted a `#` here, the SEE ALSO section always has three `###`.

Reply:
You are correct.  I've pushed an update to address that